### PR TITLE
Multiuser

### PR DESCRIPTION
**Pull request by adambedford**
Made some changes based on comments from @einzige

### CODE REVIEW

**Comment by cantino** [comment]
Is this working for you now @adambedford?

File: lib/mavenlink/model.rb [hunk 1]
```diff
@@ -36,6 +38,11 @@ def self.scoped
       Mavenlink::Request.new(collection_name)
     end
 
+    # @return [Mavenlink::Request]
+    def scoped_im
```
Comment (einzige) [inline lib/mavenlink/model.rb]: remove this from public

**Comment by einzige** [comment]
am i the only one who feels a lack of tests for this? let's directly mock requests for 2 hosts and see returning records from multiple clients

File: spec/lib/mavenlink/workspace_spec.rb [hunk 2]
```diff
@@ -205,6 +213,7 @@
     it 'accepts attributes' do
       expect(model.new(any_custom_key: 'value set')).to include(any_custom_key: 'value set')
     end
+
```
Comment (einzige) [inline spec/lib/mavenlink/workspace_spec.rb]: extra line

File: spec/lib/mavenlink/model_spec.rb [hunk 3]
```diff
@@ -175,6 +175,10 @@ def self.name
     it 'accepts attributes' do
       expect(model.new(any_custom_key: 'value set')).to include(any_custom_key: 'value set')
     end
+
+    it 'checks for client' do
```
Comment (einzige) [inline spec/lib/mavenlink/model_spec.rb]: for such simple tests the following would give more descriptive output: 
```ruby
its(:client) { should be_a Mavenlink::Client }
```

**Comment by adambedford** [comment]
@cantino yes this is working well. I should have some time early next week to work on this again.

**Comment by cantino** [comment]
Great, let us know if you need a hand!

**Comment by einzige** [comment]
@cantino please close this one

**Pull request closed without merging**